### PR TITLE
refactor: use file structs

### PR DESCRIPTION
**Pull request by tensorush**
- made use of file structs to avoid redundant namespaces
- added the `simple_spacecraft_orientation` to run with other examples in `build.zig`
- consider using [doctests](https://ziglang.org/documentation/0.13.0/#Doctests), I only changed the `CelestialBody` one to demonstrate.

### CODE REVIEW

File: src/constants.zig [hunk 1]
```diff
@@ -65,6 +65,6 @@ pub const uranus = CelestialBody.init(8.68192e25, 5.79456e6, 4.36039e-5, 25362,
 pub const neptune = CelestialBody.init(1.02431e26, 6.83653e6, 5.14447e-5, 24622, 24764, 24341, 4.50489e9, 4.46384e9, 4.54594e9, 60357.05, 5.43, 0.00911, 1.77, 0.01708, 0.003411, 0.45, 19.7);
 pub const pluto = CelestialBody.init(1.46158e22, 9.75500e2, 7.34061e-9, 1188.3, null, null, 5.91540e9, 4.44212e9, 7.38868e9, 90821.51, 4.74, 0.24906, 17.14, 0.002, 0.00039, 1e-6, 50.0);
 
-test "Test Celestial Bodies Made" {
+test CelestialBody {
```
Comment (tensorush) [inline src/constants.zig:68]: Just use the name of the tested entity, ie function, struct, etc.

**Comment by ATTron** [comment]
thanks for this! I will get to this today and look it over :star:

**Comment by ATTron** [comment]
This all looks good to me! I might go back at some point and switch Ecs back to EquatorialCoordinateSystem, just for me I like having the full names of things when I can. If there is some zig style guide opinions about this I will defer to them, but my own personal preference is to have the full name be the structs name

**Comment by tensorush** [comment]
Sure, I think long names are fine, I changed it back. My reasoning was that it seemed the odd one out as it's comprised of 3 pretty long words, so I chose to use its acronym in code, while having the full words in the doc comment above its definition.

**Pull request merged**